modules/ob-compatibility-support-matrix.adoc

```diff
@@ -23,6 +23,6 @@ The link:https://access.redhat.com/support/offerings/techpreview[Technology Prev
 Openshift Pipelines Version | OpenShift Version
 
 | Operator | Builds (Shipwright) | CLI | |
-|1.0 | 0.12.0 (GA) | 0.12.0 (GA) | 1.12 and later  | 4.14
+|1.0 | 0.13.0 (GA) | 0.13.0 (GA) | 1.15 and later  | 4.16
```

Comment:
```suggestion
|1.0 | 0.12.0 (GA) | 0.12.0 (GA) | 1.12 and later  | 4.14
|1.1 | 0.13.0 (GA) | 0.13.0 (GA) | 1.15 and later  | 4.16
```
A new section for builds v1.1 compatibility should be added. And the compatibility of v1.0 remains as in the [previous release](https://docs.openshift.com/builds/1.0/about/ob-release-notes.html).